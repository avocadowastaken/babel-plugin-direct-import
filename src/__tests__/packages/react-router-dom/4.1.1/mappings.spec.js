const { runMappingSpec } = require('../../../../__testutils__/run-spec');

runMappingSpec(__dirname);
