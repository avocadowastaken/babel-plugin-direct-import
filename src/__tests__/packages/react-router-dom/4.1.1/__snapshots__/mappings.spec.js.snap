// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fulfill mappings for: react-router-dom@4.1.1 should fulfill exports for "react-router-dom/es/index" 1`] = `
Object {
  "exports": Object {
    "BrowserRouter": Object {
      "exported": "BrowserRouter",
      "local": "default",
      "source": "react-router-dom/es/BrowserRouter",
    },
    "HashRouter": Object {
      "exported": "HashRouter",
      "local": "default",
      "source": "react-router-dom/es/HashRouter",
    },
    "Link": Object {
      "exported": "Link",
      "local": "default",
      "source": "react-router-dom/es/Link",
    },
    "MemoryRouter": Object {
      "exported": "MemoryRouter",
      "local": "default",
      "source": "react-router-dom/es/MemoryRouter",
    },
    "NavLink": Object {
      "exported": "NavLink",
      "local": "default",
      "source": "react-router-dom/es/NavLink",
    },
    "Prompt": Object {
      "exported": "Prompt",
      "local": "default",
      "source": "react-router-dom/es/Prompt",
    },
    "Redirect": Object {
      "exported": "Redirect",
      "local": "default",
      "source": "react-router-dom/es/Redirect",
    },
    "Route": Object {
      "exported": "Route",
      "local": "default",
      "source": "react-router-dom/es/Route",
    },
    "Router": Object {
      "exported": "Router",
      "local": "default",
      "source": "react-router-dom/es/Router",
    },
    "StaticRouter": Object {
      "exported": "StaticRouter",
      "local": "default",
      "source": "react-router-dom/es/StaticRouter",
    },
    "Switch": Object {
      "exported": "Switch",
      "local": "default",
      "source": "react-router-dom/es/Switch",
    },
    "matchPath": Object {
      "exported": "matchPath",
      "local": "default",
      "source": "react-router-dom/es/matchPath",
    },
    "withRouter": Object {
      "exported": "withRouter",
      "local": "default",
      "source": "react-router-dom/es/withRouter",
    },
  },
  "indexFile": "react-router-dom/es/index",
  "name": "react-router-dom",
}
`;
